WordPress/Classes/ViewRelated/Stats/SiteStatsTableViewCells.swift

```diff
@@ -206,10 +158,18 @@ struct TopTotalsInsightStatsRow: ImmuTableRow {
             return
         }
 
+        let limitRowsDisplayed: Bool = {
+            if let statSection = dataRows.first?.statSection, statSection == .insightsPublicize {
+                return false
+            }
+            return true
```

Comment:
What do you think if this can be reduced to:
`return !(dataRows.first?.statSection == .insightsPublicize)`

Reply:
Ah, yes, that is much cleaner. It shall be done. Thanks!